terraform/deployments/cluster-services/argo.tf

```
@@ -76,76 +78,45 @@ resource "helm_release" "argo_notifications" {
   repository = "https://argoproj.github.io/argo-helm"
   version    = "1.5.1" # TODO: Dependabot or equivalent so this doesn't get neglected.
   values = [yamlencode({
-    argocdUrl = "https://${local.argo_host}"
+    # Configured in argo-services Helm chart
+    "cm.create" = false
+    "argocdUrl" = "https://${local.argo_host}"
 
     # argocd-notifications-secret will be created by ExternalSecrets
     # since the secrets are stored in AWS SecretsManager
-    secret = {
-      create = false
-    }
+    "secret.create" = false
+  })]
+}
 
-    notifiers = {
-      # this slack webhook (managed by IT Services) allows messages on the `govuk-deploy-alerts` channel 
-      "service.webhook.slack_webhook" = yamlencode({
-        url = "$slack_url"
-        headers = [{
-          name  = "Content-Type"
-          value = "application/json"
-        }]
-      })
-      "service.slack" = null # remove default unconfigured slack service to remove misconfiguration error
+resource "helm_release" "argo_workflows" {
+  chart      = "argo-workflows"
+  name       = "argo-workflows"
+  namespace  = local.services_ns
+  repository = "https://argoproj.github.io/argo-helm"
+  version    = "0.8.0" # TODO: Dependabot or equivalent so this doesn't get neglected.
+  values = [yamlencode({
+    controller = {
+      workflowNamespaces = [local.services_ns, local.apps_ns]
     }
 
-    triggers = {
-      "trigger.sync-operation-change" = yamlencode([
-        {
-          when    = "app.status.operationState.phase in ['Succeeded'] and app.status.health.status == 'Healthy'"
-          oncePer = "app.status.sync.revision"
-          send    = ["send-slack"]
-        },
-        {
-          when    = "app.status.operationState.phase in ['Running']"
-          oncePer = "app.status.sync.revision"
-          send    = ["send-slack"]
-        },
-        {
-          when    = "app.status.operationState.phase in ['Error', 'Failed']"
-          oncePer = "app.status.sync.revision"
-          send    = ["send-slack"]
-        },
-      ])
+    workflow = {
+      serviceAccount = {
+        create = true # Specifies whether a service account should be created
```

Please remove redundant comment
```suggestion
        create = true
```